demos/serverless/src/ChimeMeetingsWrapper.js

```diff
@@ -0,0 +1,60 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+const AWS = require('aws-sdk');
+
+module.exports = class ChimeMeetingsWrapper {
```

Comment:
This looks like a dupe file.  Is there a way we can have both the browser and the serverless demos reference the same file?

Reply:
@aws-geoff Actually we need two separate files as these files need to access aws-sdk from node folders and both the browser and serverless demo has different package.json and thus sepearate node folders.